pkg/private/tar/tar_writer.py

```diff
@@ -0,0 +1,369 @@
+# Copyright 2022 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Archive manipulation library for the Docker rules."""
+
+# pylint: disable=g-import-not-at-top
```

Comment:
Is this directive google-specific?  Also, is it necessary here, and if so, why?  All the imports are at the top of the file, albeit below the module docstring.

Reply:
It may have been google-specific. But certainly not needed now.